packages/ts/package.json

```diff
@@ -22,7 +22,7 @@
 		"cached-factory": "^0.1.0",
 		"debug-for-file": "^0.2.0",
 		"ts-api-utils": "^2.1.0",
-		"typescript": ">=5.9.3"
+		"typescript": "^5.9.0 || ^6.0.0"
```

Comment:
Two thoughts:
1. Should this be a peer?
2. Can we use catalogs?

Reply:
> peer

Some users won't have `typescript` installed. They could be using this on a plain-JS package, for example. So this does need to be a standard dep.

> catalogs

🤷 I've never found a need for them, and it works as-is. But if you've got a compelling case I'd be interested to hear it.